challenges/objects-arrays.js

```diff
@@ -6,29 +6,51 @@
   object name, diet, weight, length, period
 */
 
-// tyrannosaurus, carnivorous, 7000kg, 12m, Late Cretaceous
-
+const tRex = {
+  name:'tyrannosaurus',
+  diet: 'carnivorous',
+  weight: '7000kg',
+  length: '12m',
+  period: 'Late Cretaceous'
+  
+};
 // stegosaurus, herbivorous, 2000kg, 9m, Late Jurassic
-
+ const  steggy = {
+  name: 'stegosaurus',
+  diet: 'herbivorous',
+  weight: '2000kg',
+  length: '9m',
+  period: 'Late CJurassic'
+ };
 // velociraptor, carnivorous, 15kg, 1.8m, Late Cretaceous
-
-// Using your dinosaur objects, log answers to these questions:
+ const vRaptor = {
+  name: 'velociraptor',
+  diet: 'carnivorous',
+  weight: '15kg',
+  length: '1.8m',
+  period: 'Late Jurassic'
+ };
```

Comment:
This is what having those inheritance chains replaces (whether functional && prototypical inheritance or Class based object construction.)